#Message where final commands for the MQS are concatenated to

uint8[16] cmds
